pkgs/servers/sip/freeswitch/default.nix

```diff
@@ -110,6 +126,15 @@ stdenv.mkDerivation rec {
     done
   '';
 
+  ## TODO Validate with the next upstream release
+  pathces = [
+    (fetchpatch {
+       name = "CVE-2023-5217.patch";
+       url = "https://github.com/signalwire/freeswitch/commit/f1fb05214e4f427dcf922f531431ab649cf0622b.patch";
```

Comment:
I think you want https://github.com/signalwire/freeswitch/commit/6f9e72c585265d8def8a613b36cd4f524c201980 for CVE-2023-5217.

We also want this patch but it is for CVE-2023-44488.

Reply:
Oh, I see! I'll fix it right away. Just waiting to see if it builds (:

Reply:
@LeSuisse Have added both patches now and tested that it builds.